Utils/Elasticsearch/requests/dataset_aggregations.json

```diff
@@ -0,0 +1,155 @@
+/*
+  Aggregated summary:
+  Average not deleted dataset size, categorized by campaigns and output formats
+
+  There are many occasions when aggregations are required but search hits are not.
+  For these cases the hits can be ignored by setting size=0.
+*/
+GET prodsys_rucio_ami/task/_search
+{
+   "size": 0,
+   "aggs": {
+     "campaign": {
+       "terms": {
+         "field": "campaign.keyword"
+       },
+       "aggs": {
+         "format": {
+            "terms": {
+              "field": "output_formats"
+            },
```

Comment:
Currently this request returns average dataset size for all cases of 'output_formats', which is processing like string. When we implement new mapping, with 'pattern' analyzer - this request will be changed to group dataset formats, according to 'datasetname' value.

Reply:
So, we should not split this dot-splitted list into real list on Stage 016?

Reply:
I have to think about this field. I'll try to answer tomorrow ))

Reply:
I suppose we should split output_formats by dot in Stage 016! 
But we wouldn't make aggregations on this field, as this is just enumeration on output data formats for the task. 
If we need aggregation by data format we will use 'data_format' parameter from 'output_dataset' type. 
'output_formats' will be used only, for example, to search all tasks, returning 'DAOD_*' data format, or all tasks, returning both 'AOD' and 'HITS' data formats.